src/main/java/FC/Frame.java

```diff
@@ -399,22 +394,62 @@ private void comboBoxClicked() {
         topPanel.setVisible(true);
         editPanel.setVisible(false);
         deleteDeckPanel.setVisible(false);
+    }
 
+    private void setupStudyMode() {
+        studyPanel = new JPanel();
+        studyButtonPanel = new JPanel(new FlowLayout()); //just for buttons
+        studyPanel.setLayout(new BoxLayout(studyPanel, BoxLayout.Y_AXIS));
+        studyPanel.setBorder(new EmptyBorder(200, 0, 0, 0));
+        termTextArea = new JLabel();//text starts in middle
+        termTextArea.setLayout(new FlowLayout());
+        defTextArea = new JLabel();
```

Comment:
WIP: added this but didn't finish. Need to fix the issue of definition starting in the middle